src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputMethodParamPadCheckRecordPattern.java

```diff
@@ -0,0 +1,53 @@
+/*
+MethodParamPad
+allowLineBreaks = true
+option = (default)nospace
+tokens = (default)CTOR_DEF, LITERAL_NEW, METHOD_CALL, METHOD_DEF, SUPER_CTOR_CALL, \
+         ENUM_CONSTANT_DEF, RECORD_DEF, RECORD_PATTERN_DEF
+
+
+*/
+
+//non-compiled with javac: Compilable with Java21
+package com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad;
+
+public class InputMethodParamPadCheckRecordPattern {
+
+    void test(Object obj) {
+
+        // violation below, ''(' is preceded with whitespace'
+        if (obj instanceof ColoredPoint (Point p, String s)) {
```

Comment:
Also show examples with space after `(`. Like https://checkstyle.org/property_types.html#PadOption has.

Reply:
done. but for clarity, the spaces after `(` is a concern for `ParenPad`.
 `MethodParamPad` is concerned with the space before ` ( `